packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -346,7 +344,7 @@ export async function load_node({
 				return proxy;
 			},
 			stuff: { ...stuff },
-			status: is_error ? status ?? null : null,
+			status: is_error ? status ?? null : shadow.status ?? null,
```

Comment:
how do you mean — like this?

```suggestion
			status: (is_error ? status : shadow.status) ?? null,
```

Reply:
That's what I had meant, yeah. I think it's a bit easier on the eyes, but I don't feel strongly about it.

I am still curious about needing `?? null` at all though, and what would behave differently if `status` were `undefined` rather than `null`.